flexx/app/_tornadoserver.py

```diff
@@ -676,15 +676,18 @@ def check_origin(self, origin):
         #WebSocketHandler.check_origin
         
         serving_host = self.request.headers.get("Host")
-        serving_hostname = serving_host.split(':')[0]
-        connecting_host = urlparse(origin).netloc
+        serving_hostname = serving_host.split(':')[0].lower()
+        connecting_host = urlparse(origin).netloc.lower()
+
         
         if serving_hostname == 'localhost':
             return True  # Safe
         elif serving_hostname == '0.0.0.0':
             return True  # we cannot know if the origin matches
```

Comment:
A socket (i.e. server) can bind to `0.0.0.0` which means that any connection is accepted that is targeting the machine (e.g. 'localhost' works, but also the IP for the LAN connection and the IP for the WLAN connection).

Anyway, I'll merge this now. I'm planning a few tweaks areound `flexx.app` anyway.